cli/types/index.d.ts

```diff
@@ -75,7 +75,7 @@ declare namespace Cypress {
     (task: 'firefox:force:gc'): Promise<void>
   }
 
-  type BrowserName = 'electron' | 'chrome' | 'chromium' | 'firefox' | string
+  type BrowserName = 'electron' | 'chrome' | 'chromium' | 'firefox' | 'edge' | 'brave' | string

```

Comment:
i think the enumerated BrowserNames should be limited to the ones we support out of the box, so:

```suggestion
  type BrowserName = 'electron' | 'chrome' | 'chromium' | 'firefox' | 'edge' | string

```

Reply:
so no `| string` ?

Reply:
the `| string` can stay, so we can have correct types for user-supplied browsers